vnext/Shared/OInstance.cpp

```diff
@@ -418,6 +422,25 @@ InstanceImpl::InstanceImpl(
           m_devSettings->jsiRuntimeHolder =
               std::make_shared<Microsoft::JSI::ChakraRuntimeHolder>(m_devSettings, m_jsThread, nullptr, nullptr);
           break;
+        case JSIEngineOverride::V8Napi: {
+#if defined(USE_V8)
+          std::unique_ptr<facebook::jsi::ScriptStore> scriptStore = nullptr;
+          std::unique_ptr<facebook::jsi::PreparedScriptStore> preparedScriptStore = nullptr;
+
+          char tempPath[MAX_PATH];
+          if (GetTempPathA(MAX_PATH, tempPath)) {
```

Comment:
>GetTempPathA [](http://example.com/codeflow?start=14&length=12)

This will fail on Unicode paths, please use GetTempPathW

Reply:
Note, the "normal" V8 case also uses `GetTempPathA`.
I'll change this in both.